src/pages/course-page.js

```diff
@@ -74,6 +74,13 @@ const CoursePage = (props) => {
     return `${date}/${month}/${year}`;
   };
 
+  const checkAuthor = (review) => {
```

Comment:
function name: not really checking, but displaying the author so a better function name would be `showAuthor` or `displayAuthor` 

also since it's a one if block statement, we can do a ternary if statement for cleaner code:

```
return review.displayAuthor ? review.author : 'Anonymous'
```